### DISCUSSION TITLE
Centerpieces > Tool_hooks > 2x5 Irwin pliers holders help

### DISCUSSION THREAD

**Discussion by nullexxx** [aderusha/DDD-Printable-Wall-Control-System#65] [Q&A] [answered]
hello,

I can't for the life of my figure out which sidepieces go with these items..  can someone please help? 

Thanks

**Comment by razorfrog** [comment, marked as answer]
Anything that sticks out 2 and then a support across the bottom. I would use these: 
- [2x2 Angle Bracket Flat Left.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Sidepieces/Angle_brackets/2x2%20Angle%20Bracket%20Flat%20Left.stl)
- [2x2 Angle Bracket Flat Right.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Sidepieces/Angle_brackets/2x2%20Angle%20Bracket%20Flat%20Right.stl)
- [2x5 Locking Spacer.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Centerpieces/Locking_spacer/2x5%20Locking%20Spacer.stl)


Tool files: 
- [2x5 Irwin Pliers Crimper Stripper.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Centerpieces/Tool_hooks/2x5%20Irwin%20Pliers%20Crimper%20Stripper.stl)
- [2x5 Irwin Pliers LgLock Linemans.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Centerpieces/Tool_hooks/2x5%20Irwin%20Pliers%20LgLock%20Linemans.stl)
- [2x5 Irwin Pliers LgNeedle AviationSnips.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Centerpieces/Tool_hooks/2x5%20Irwin%20Pliers%20LgNeedle%20AviationSnips.stl)
- [2x5 Irwin Pliers SmNeedle SAWireStripper.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Centerpieces/Tool_hooks/2x5%20Irwin%20Pliers%20SmNeedle%20SAWireStripper.stl)

**Reply by nullexxx** [reply]
You're wonderful, thank you!

**Reply by razorfrog** [reply]
Although, because of the extra height of these, you might want to use the reversed side pieces instead: 
- [2x2 Angle Bracket Inverted Flat Left.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Sidepieces/Angle_brackets/2x2%20Angle%20Bracket%20Inverted%20Flat%20Left.stl)
- [2x2 Angle Bracket Inverted Flat Right.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Sidepieces/Angle_brackets/2x2%20Angle%20Bracket%20Inverted%20Flat%20Right.stl)

**Reply by nullexxx** [reply]
> Although, because of the extra height of these, you might want to use the reversed side pieces instead:
> 
>     * [2x2 Angle Bracket Inverted Flat Left.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Sidepieces/Angle_brackets/2x2%20Angle%20Bracket%20Inverted%20Flat%20Left.stl)
> 
>     * [2x2 Angle Bracket Inverted Flat Right.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Sidepieces/Angle_brackets/2x2%20Angle%20Bracket%20Inverted%20Flat%20Right.stl)

yea, I had originally tried the inverted ones first and they didn't look right, which is why I figured I would ask here before I printed anything else for them. I'll give the regular ones a shot and see how they look.

Thanks again!
